src/main/webapp/firebase-chat.js

```diff
@@ -0,0 +1,280 @@
+/* eslint-disable */
+const firebaseConfig = {
+  // apiKey: removed ,
+  authDomain: 'lounge-95f01.firebaseapp.com',
+  databaseURL: 'https://lounge-95f01.firebaseio.com',
+  projectId: 'youtube-lounge',
+  storageBucket: 'youtube-lounge.appspot.com',
+  messagingSenderId: '681171972170',
+  appId: '1:681171972170:web:4c6526b8eb788af9d876b3',
+  measurementId: 'G-JSDHBSMHS3',
+};
+firebase.initializeApp(firebaseConfig); // eslint-disable-line no-undef
```

Comment:
Looks like this works without the import lint errors you were seeing previously.